src/Bicep.Core/Analyzers/Linter/Rules/EnvironmentUrlHardcodedRule.cs

```diff
@@ -35,47 +35,57 @@ public override void Configure(IConfigurationRoot config)
         }
 
         protected override string FormatMessage(params object[] values)
-            => string.Format("{0} -- Found: [{1}]", this.Description, values.First());
+            => string.Format("{0} -- Found: '{1}'", this.Description, values.First());
```

Comment:
>' [](http://example.com/codeflow?start=44&length=1)

We've actually standardized on `"` as the quote char for the diagnostic messages.

Reply:
=> string.Format("{0} Found this disallowed host: \"{1}\"", this.Description, values.First());